scripts/sysman/firmware.yml

```diff
@@ -90,6 +90,43 @@ returns:
     - $X_RESULT_ERROR_INSUFFICIENT_PERMISSIONS:
         - "User does not have permissions to perform this operation."
 --- #--------------------------------------------------------------------------
+type: function
+desc: "Get Firmware Flash Progress"
+version: "1.8"
+class: $sFirmware
+name: GetFlashProgress
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_firmware_handle_t
+      name: hFirmware
+      desc: "[in] Handle for the component."
+    - type: "uint32_t*"
+      name: pCompletionPercent
+      desc: "[in,out] Pointer to the Completion Percentage of Firmware Update"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get Firmware Logs"
```

Comment:
desc to be updated as well to "Get Firmware Console Logs" ?

Reply:
Done